python/tank/authentication/login_dialog.py

```diff
@@ -277,9 +279,8 @@ def _get_current_site(self):
 
         :returns: The site to connect to.
         """
-        return six.ensure_str(
-            connection.sanitize_url(self.ui.site.currentText().strip())
-        )
+        url = self.ui.site.currentText().strip()
+        return six.ensure_str(connection.sanitize_url(url))
```

Comment:
The code hasn't changed here, just reformatted. But something important has changed here under the hood. In the __init__ method, we call this method to know if there is a current site or not. If a site is set, we will focus the user of password field accordingly, depending on which needs to be filled in yet. However, this causes a problem because sanitize_url used to return `https://`when the host was empty, which meant that we could the focus the login or password box instead of the host. Now that sanitize_url does not prefix https:// to an empty url,the __init__ now properly puts focus on the host field.